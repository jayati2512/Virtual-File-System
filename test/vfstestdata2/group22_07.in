createvfs demodemodemodemodemodemodemodemodemo07 10
mountvfs demodemodemodemodemodemodemodemodemo07
makedir /home/ dir1
createvfs demo07 20
mountvfs demo07
makedir /home/ dir/1
makedir /home/dir1/ dir2
addfile /home/ file1.txt /usr/include/zlibdefs.h
eportfile /home/ /tmp/export1
exportfile /home/file1.txt /tmp/export1
listfile /home/file2.txt /tmp/listfile1
updatefile /home/file1.txt /usr/include/zlibdefs.h
listfile /home/file1.txt /tmp/listfile1
mountvfs demo07
unmountvfs demo07
movedir /home/dir1/ /
