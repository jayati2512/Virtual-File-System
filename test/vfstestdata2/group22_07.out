VFS_LABEL_TOO_LARGE
DATA_FILE_NOT_FOUND
VFS_NOT_MOUNTED
createvfs_SUCCESS
mountvfs_SUCCESS
INVALID_CHARACTER_IN_DIRNAME
makedir_SUCCESS
addfile_SUCCESS
Ignoring invalid command eportfile
exportfile_SUCCESS
SOURCE_FILE_PATH_NOT_FOUND
updatefile_SUCCESS
listfile_SUCCESS
VFS_ALREADY_MOUNTED
unmountvfs_SUCCESS
VFS_NOT_MOUNTED
